content/en/events/2024/kcseu/schedule.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Schedule
+type: docs
+weight: 3
+---
+
+**NOTE:** You **MUST** pick up your badge before attending the summit or social evening event.
+
+## Steering AMA
+
+The Summit will start off with an AMA with members of the Kubernetes Steering
+Committee.  Bring any and all questions about how we're running the project,
+ongoing efforts & issues, and whatever you'd like to discuss with Steering.
+
+## Unconference
+
+TBD
+
+## Doc Sprint
+
+SIG-Docs will have a full-day Documentation working session during the Summit.
+If you contribute to the docs -- and who doesn't -- or you want to learn to
+do it better, join them.
+
+## CFP
+
+If you would like to submit a formal session, please use the [CFP Form]().
```

Comment:
Remove Form for now and Replace with:

The CFP will open soon and will cover presentation, workshop, and collaboration style sessions.  
Unconference sessions should use the GitHub issue referenced above.